src/current/v23.2/cloud-storage-authentication.md

```diff
@@ -388,17 +405,22 @@ For CockroachDB clusters running in other environments, `implicit` authenticatio
 
 {% include {{ page.version.version }}/backups/external-io-implicit-flag.md %}
 
-## Google Cloud Storage assume role
+<a id="use-implicit-authentication-dedicated-gcp">
+### Use implicit authentication with CockroachDB {{ site.data.products.dedicated }}
 
-{{site.data.alerts.callout_info}}
-CockroachDB supports assume role authentication on clusters running v22.2. Authenticating to cloud storage with `ASSUME_ROLE` on clusters running versions v22.1 and earlier, or mixed versions, is not supported and will result in failed bulk operations.
-{{site.data.alerts.end}}
+For a CockroachDB {{ site.data.products.dedicated }} cluster, you can either [use implicit authentication with your own service account](#use-implicit-authentication-with-your-own-service-account) or you can follow these instructions to use implicit authentication with the service account that CockroachDB {{ site.data.products.cloud }} automatically creates and manages for each CockroachDB {{ site.data.products.dedicated }} cluster. This service account is named according to the pattern `crl-dr-store-user-{CLUSTER_ID_SUFFIX}`, where `{CLUSTER_ID}` is the last 12 digits of the cluster's. This service account is automatically granted the permission to write to the cluster's storage. One benefit of this approach is that no additional credentials needs to be added to the cluster.
```

Comment:
nit: not sure if you wanted to include `@{PROJECT_ID}.iam.gserviceaccount.com` in this text considering you have the full ARN for the AWS example.